aten/src/ATen/templates/DispatchKeyFunctions_inl.h

```diff
@@ -0,0 +1,16 @@
+// ${generated_comment}
+
+// NB: The implementing C++ file is RegisterDispatchKey.cpp
+
+// The only #includes we need are for custom classes that have defaults in the C++ API
+#include <c10/core/MemoryFormat.h>
+#include <c10/core/Scalar.h>
+#include <ATen/core/Reduction.h>
```

Comment:
doesn't look like git displayed the [rename `DispatchKeyFunctions.h` -> `DispatchKeyFunctions_inl.h`] + [create `DispatchKeyFunctions.h`] changes nicely. The only real change was tightening the #includes.